### ISSUE TITLE
Could not load plugin vcv-link-master - plugin.dylib doesn't exist

### ISSUE THREAD

**Issue by aitorgrande** [stellare-modular/vcv-link#4]
I cannot install the plugin in latest version of vcv rack. This is the log

[0.000 info src/main.cpp:67] VCV Rack 0.6.2c
[0.000 info src/main.cpp:70] Global directory: /Applications/Rack.app/Contents/Resources/
[0.000 info src/main.cpp:71] Local directory: /Users/aitorgrande/Documents/Rack/
[0.000 info src/plugin.cpp:303] Extracting package /Users/aitorgrande/Documents/Rack/plugins/ML_modules.zip
[0.187 info src/plugin.cpp:303] Extracting package /Users/aitorgrande/Documents/Rack/plugins/SonusModular.zip
[0.450 info src/plugin.cpp:303] Extracting package /Users/aitorgrande/Documents/Rack/plugins/dBiz.zip
[1.125 info src/plugin.cpp:303] Extracting package /Users/aitorgrande/Documents/Rack/plugins/JW-Modules.zip
[1.722 info src/plugin.cpp:303] Extracting package /Users/aitorgrande/Documents/Rack/plugins/Grayscale.zip
[1.784 info src/plugin.cpp:303] Extracting package /Users/aitorgrande/Documents/Rack/plugins/ESeries.zip
[1.787 info src/plugin.cpp:303] Extracting package /Users/aitorgrande/Documents/Rack/plugins/AmalgamatedHarmonics.zip
[1.915 info src/plugin.cpp:125] Loaded plugin Grayscale 0.6.0 from /Users/aitorgrande/Documents/Rack/plugins/Grayscale/plugin.dylib
[1.990 info src/plugin.cpp:125] Loaded plugin Befaco 0.6.0 from /Users/aitorgrande/Documents/Rack/plugins/Befaco/plugin.dylib
[1.991 info src/plugin.cpp:125] Loaded plugin AmalgamatedHarmonics 0.6.3.1 from /Users/aitorgrande/Documents/Rack/plugins/AmalgamatedHarmonics/plugin.dylib
[1.993 info src/plugin.cpp:125] Loaded plugin dBiz 0.6.1 from /Users/aitorgrande/Documents/Rack/plugins/dBiz/plugin.dylib
[1.995 info src/plugin.cpp:125] Loaded plugin SonusModular 0.6.2 from /Users/aitorgrande/Documents/Rack/plugins/SonusModular/plugin.dylib
**[1.995 warn src/plugin.cpp:73] Plugin file /Users/aitorgrande/Documents/Rack/plugins/vcv-link-master/plugin.dylib does not exist**
[1.996 info src/plugin.cpp:125] Loaded plugin JW-Modules 0.6.3 from /Users/aitorgrande/Documents/Rack/plugins/JW-Modules/plugin.dylib
[2.001 info src/plugin.cpp:125] Loaded plugin Fundamental 0.6.2 from /Users/aitorgrande/Documents/Rack/plugins/Fundamental/plugin.dylib
[2.002 info src/plugin.cpp:125] Loaded plugin ESeries 0.6.0 from /Users/aitorgrande/Documents/Rack/plugins/ESeries/plugin.dylib
[2.010 info src/plugin.cpp:125] Loaded plugin AudibleInstruments 0.6.3 from /Users/aitorgrande/Documents/Rack/plugins/AudibleInstruments/plugin.dylib
[2.019 info src/plugin.cpp:125] Loaded plugin 21kHz 0.6.1 from /Users/aitorgrande/Documents/Rack/plugins/21kHz/plugin.dylib
[2.021 info src/plugin.cpp:125] Loaded plugin ML_modules 0.6.3 from /Users/aitorgrande/Documents/Rack/plugins/ML_modules/plugin.dylib
[101.600 info src/bridge.cpp:339] Bridge server started
[101.911 info src/bridge.cpp:106] Bridge client connected
[102.079 info src/window.cpp:642] Loaded font /Applications/Rack.app/Contents/Resources/res/fonts/DejaVuSans.ttf
[102.079 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/icons/noun_146097_cc.svg
[102.079 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/icons/noun_31859_cc.svg
[102.079 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/icons/noun_1343816_cc.svg
[102.080 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/icons/noun_1343811_cc.svg
[102.080 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/icons/noun_1084369_cc.svg
[102.080 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/icons/noun_1745061_cc.svg
[102.080 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/icons/noun_1240789_cc.svg
[102.081 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/icons/noun_305536_cc.svg
[102.081 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/icons/noun_468341_cc.svg
[102.081 info src/settings.cpp:172] Loading settings /Users/aitorgrande/Documents/Rack/settings.json
[102.087 info src/settings.cpp:158] Saving settings /Users/aitorgrande/Documents/Rack/settings.json
[102.087 info src/app/RackWidget.cpp:143] Loading patch /Users/aitorgrande/Documents/Rack/autosave.vcv
[102.164 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/Core/AudioInterface.svg
[102.164 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/ComponentLibrary/ScrewSilver.svg
[102.168 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/ComponentLibrary/PJ301M.svg
[102.168 info src/window.cpp:642] Loaded font /Applications/Rack.app/Contents/Resources/res/fonts/ShareTechMono-Regular.ttf
[102.176 info src/window.cpp:695] Loaded SVG /Users/aitorgrande/Documents/Rack/plugins/Befaco/res/EvenVCO.svg
[102.176 info src/window.cpp:695] Loaded SVG /Users/aitorgrande/Documents/Rack/plugins/Befaco/res/Knurlie.svg
[102.176 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/ComponentLibrary/BefacoBigKnob.svg
[102.177 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/ComponentLibrary/BefacoTinyKnob.svg
[102.177 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/ComponentLibrary/Davies1900hRed.svg
[102.178 info src/window.cpp:695] Loaded SVG /Applications/Rack.app/Contents/Resources/res/Core/MIDIToCVInterface.svg
[102.270 debug src/Core/AudioInterface.cpp:74] Audio Interface IO underflow
[102.384 debug src/Core/AudioInterface.cpp:74] Audio Interface IO underflow

**Comment by JeffGreenlee42** [comment]
Ditto..   Rack 6.2c will not load this module.

**Comment by aitorgrande** [comment]
> Ditto.. Rack 6.2c will not load this module.

How can i download the previous version? I downloaded the 6.2b from github but cannot install it

**Comment by JeffGreenlee42** [comment]
Well.. Turns out I had to re-install it.. Had to use the version 6.1  of
the Stellare link module.  Turns out I didn't have to do any compiling or
anything (thought I would). just unzip itg and put it in the Plugins
folder.
 I have no clue why the version I had suddenly would not cooperate.  It
happened after I upgraded to Rack 6.2c .   I removed Stellare-Link and then
installed it again.. now it works find.


Jeff Greenlee

"Music is a higher revelation than all wisdom and philosophy. Music is the
electrical soil in which the spirit lives, thinks and invents."
Ludwig Van Beethoven
